sox.features.js

```diff
@@ -1098,58 +1086,58 @@
 
         // Make diamond blue if there's a new question
         if (latestQuestion != lastQuestions[metaName]) {
-          $diamond.css('color', '#0077cc');
+          diamond.style.color = '#0077cc';
         }
 
         let seen = false;
         // Regardless of if there's a new question, show the latest 5
-        for (let i = 0, len = items.length; i < len; i++) {
-          const title = items[i].title;
-          const link = items[i].link;
+        items.forEach(item => {
+          const title = item.title;
+          const link = item.link;
           // If one's been seen, the older questions must also have been seen
           if (title === lastQuestions[metaName]) seen = true;
           addQuestion(title, link, seen);
-        }
+        });
 
         lastQuestions[metaName] = latestQuestion;
 
-        $diamond.click(() => {
-          GM_setValue(NEWQUESTIONS, JSON.stringify(lastQuestions));
-        });
+        diamond.addEventListener('click', () => GM_setValue(NEWQUESTIONS, JSON.stringify(lastQuestions)));
       });
 
       function addQuestion(title, link, seen) {
-        const $li = $('<li/>');
-        const $link = $('<a/>', {
-          href: link,
-          style: 'display: flex',
-        });
-        const $icon = $('<div/>', {
-          'class': 'site-icon favicon ' + favicon,
-          style: 'margin-right: 3px',
-        });
-        const $message = $('<div/>', {
-          'class': 'message-text',
-        }).append($('<h4/>', {
-          html: title,
-          style: `font-weight: ${seen ? 'normal' : 'bold'}`,
-        }));
+        const li = document.createElement('li');
+        const $link = document.createElement('a');
```

Comment:
Could you remove the `$` here? it might be confusing when reading thinking it is a jQuery element but actually isn't!